packages/discovery-provider/src/tasks/calculate_trending_challenges.py

```diff
@@ -73,7 +77,11 @@ def dispatch_trending_challenges(
 
 
 def enqueue_trending_challenges(
-    db: SessionManager, redis: Redis, challenge_bus: ChallengeEventBus, date: date
+    db: SessionManager,
+    web3: Web3,
+    redis: Redis,
+    challenge_bus: ChallengeEventBus,
+    date: date,
```

Comment:
we're doing cooldown for trending too? aren't those auto-disbursed?
this code is fine either way just asking.

Reply:
ooh yea good point, it's not cooldown yet but maybe good to have this as a standard everywhere in case